### PR TITLE
GUIDE: Zram swap space

### PR DESCRIPTION
**Pull request by ric96**

### CODE REVIEW

**Comment by Mani-Sadhasivam** [comment]
@ric96 Using zram requires the support to be enabled in kernel. According to this guide, zram support should be built as a module, which may not be present for all CE boards by default. So, please add a section for enabling zram support (Kconfig option is enough) in kernel and mention that it is enabled by default in db410c images.

**Comment by ric96** [comment]
@Mani-Sadhasivam  ok, I've added a line mentioning that

File: ConsumerEdition/guides/zram_swapspace.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+It is not as fast as actual RAM, but it is close enough for the speed degradation to not be noticable.
+
+## Hardware
+
+- 96Boards Consumer Edition board (tested with DragonBoard410c)
+
+## Software
+
+- Linaro Debian Operating system snapshot Builds (http://builds.96boards.org/snapshots/)
+- If you are not using the DragonBoard410c, you will need to rebuild the kernel using ```CONFIG_ZRAM=m``` Kconfig option.
```
Comment (ndechesne) [inline ConsumerEdition/guides/zram_swapspace.md]: i had to do this in our defconfig to enable ZRAM:

+CONFIG_ZSMALLOC=m
+CONFIG_ZRAM=m

Comment (ric96) [reply ConsumerEdition/guides/zram_swapspace.md]: Doh! yes ofc. I forgot about that.

**Comment by Mani-Sadhasivam** [comment]
:+1:

**Comment by sdrobertw** [comment]
@ndechesne could you please approve this request so I can merge?

Thanks,

**Pull request merged**